LinkedList.c

```diff
@@ -0,0 +1,181 @@
+#include<stdio.h>
+#include<stdlib.h>
+struct NODE {
+	int d;		//integer type data  
+	struct NODE* next;		//pointer to same structure
+}*start;	//first node of the list
+
+int main() {
+	start=NULL;
+	
+	// Adding element to the list
+	addFront(23);
+    traverse();
+             
+	// Adding elements at the end
+    addLast(25);
+    addLast(46);
+    traverse();
+         
+    // Adding elements to front
+    addFront(12);
+    addFront(11);
+	traverse();

```

Comment:
Probably because you are using a combination of tabs and spaces. This would not be a problem if you use tab-only or space-only (recommended) indentation.